docs/quarantine.md

```diff
@@ -34,19 +34,18 @@ account systemic failures caused for instance by an infrastructure problem.
 
 ### Putting tests in quarantine
 
-A test must be put in quarantine when at least one of these conditions is met:
+A test must be put in quarantine when any of these conditions is met:
 * It has failed 2 times or more in the last two weeks.
 * It has failed 5 times or more in the last 3 days.
 
 A PR will be proposed on Mondays every two weeks with a batch of the tests that
-should be put in quarantine according to the data of the last two weeks. The PR
-will add the text `[QUARANTINE]` to each test's description in the code. An email
-will be sent to the owners of the suspected tests.
+met the first condition. A PR can be proposed at any time for the tests that meet
+the second condition. In both cases the PR will add the text `[QUARANTINE]` to
+each test's description in the code. An email will be sent to the owners of the
+suspected tests.
 
 Each quarantined test must have a team owner. The PR will add the text
-`[owner:@sig-{compute,network,storage}]` to each test's description. Also, a
-tracking ticket will be created in [this board] and will be assigned to a member
-of the owner team.
+`[owner:@sig-{compute,network,storage}]` to each test's description.
```

Comment:
How about we create teams for these sigs here: https://github.com/kubevirt/project-infra/blob/master/github/ci/prow-deploy/files/orgs.yaml#L651
This would have the advantage that we could assign the team directly on an issue or pr?